### PR TITLE
Add secondary menus

### PR DESCRIPTION
**Pull request by ceski-1**
EQ and Gyro for now. Maybe some changes to the Video tab in a different PR. These menus are used as a last resort, so let's resist the temptation to add a bunch of them.

https://github.com/user-attachments/assets/451f6c04-b8db-46bc-807e-b5134ca288f3

### CODE REVIEW

**Comment by fabiangreffrath** [comment]
I think secondary menus should still have a title line. Maybe in the form of a single tab title?

File: src/mn_setup.c [hunk 1]
```diff
@@ -2512,8 +2575,12 @@ static const char *curve_strings[] = {
     "2.4",    "2.5", "2.6", "2.7",     "2.8", "2.9", "Cubed"
 };
 
+static void MN_Gyro(void);
+
 static setup_menu_t gen_settings4[] = {
 
+    {"Gyro Options", S_FUNC, CNTR_X, M_SPC, .action = MN_Gyro},
```
Comment (fabiangreffrath) [inline src/mn_setup.c:2582]: BTW, is "gyro" still the best name to expose to the user? I mean, sure it's implemented with a gyroscope in the controllers, but I think I'd like e.g. "motion control" better.

Comment (rfomin) [reply src/mn_setup.c:2582]: I like “gyro” better, when I hear “motion control” I think of VR, Xbox Kinect and other irrelevant tech.

**Review by rfomin — APPROVED** [review]
Very clean, thank you!

**Comment by fabiangreffrath** [comment]
> I think secondary menus should still have a title line. Maybe in the form of a single tab title?

How about this for example?

```
--- a/src/mn_setup.c
+++ b/src/mn_setup.c
@@ -2439,6 +2439,11 @@ void MN_UpdateEqualizerItems(void)
     DisableItem(condition, eq_settings1, "snd_eq_high_cutoff");
 }
 
+static setup_tab_t equalizer_tabs[] = {
+    {"equalizer"},
+    {NULL}
+};
+
 static void MN_Equalizer(void)
 {
     SetItemOn(set_item_on);
@@ -2448,7 +2453,7 @@ static void MN_Equalizer(void)
     setup_screen = ss_eq;
     current_page = GetPageIndex(eq_settings);
     current_menu = eq_settings[current_page];
-    current_tabs = NULL;
+    current_tabs = equalizer_tabs;
     SetupMenuSecondary();
 }
 
@@ -2458,6 +2463,7 @@ void MN_DrawEqualizer(void)
 
     DrawBackground("FLOOR4_6");
     MN_DrawTitle(M_X_CENTER, M_Y_TITLE, "M_GENERL", "General");
+    DrawTabs();
     DrawInstructions();
     DrawScreenItems(current_menu);
 }
```

**Comment by ceski-1** [comment]
> How about this for example?

Something like this? https://github.com/fabiangreffrath/woof/pull/1895/commits/2d2baefb237f617279fc41f38b828404c008d504

<details><summary>screenshots</summary>

woof0000

woof0001

woof0002

woof0003

</details>

**Review by fabiangreffrath — APPROVED** [review]
Perfect, thank you!

**Pull request merged**